pxtlib/browserutils.ts

```diff
@@ -148,8 +148,9 @@ namespace pxt.BrowserUtils {
             matches = /Version\/([0-9\.]+)/i.exec(navigator.userAgent);
             // pinned web site have a different user agent
             // Mozilla/5.0 (iPhone; CPU iPhone OS 10_2_1 like Mac OS X) AppleWebKit/602.4.6 (KHTML, like Gecko) Mobile/14D27
+            // Mozilla/5.0 (iPad; CPU OS 10_3_3 like Mac OS X) AppleWebKit/603.3.8 (KHTML, like Gecko) Mobile/14G60
             if (!matches)
-                matches = /(iPod|iPhone|iPad) OS (\d+)/i.exec(navigator.userAgent);
+                matches = /(iPod|iPhone|iPad); CPU .*?OS (\d+)/i.exec(navigator.userAgent);
```

Comment:
isn't `?` redundant in the regex?

Reply:
Added to make sure it doesn't lookahead.

Reply:
doesn't lookahead needs to be in (?=OS)

Reply:
*? Quantifier — Matches between zero and unlimited times, as few times as possible, expanding as needed. 
(as few times as possible is what I was after, not the lookahead).